app/models/visitor.rb

```diff
@@ -0,0 +1,18 @@
+class Visitor < ActiveRecord::Base
+  include Person
+
+  belongs_to :visit
+  validates :visit, :rank, presence: true
```

Comment:
`rank` seems stilted in this context.  I don't have anything better to offer (`position` seems equally bad), so just flagging that I don't like it, really. :anguished: